src/elements/content-sidebar/activity-feed/task-new/__tests__/Task-test.js

```diff
@@ -78,7 +76,6 @@ describe('elements/content-sidebar/ActivityFeed/task-new/Task', () => {
 
     test('should show assignment status badges for each assignee', () => {
         const wrapper = mount(<Task currentUser={currentUser} onEdit={jest.fn()} onDelete={jest.fn()} {...task} />);
-        expect(wrapper).toMatchSnapshot();
```

Comment:
What were we testing previously with the snapshot? Should there be new assertions here?

Reply:
Not really, I copied the original test this from the old task component, which used a snapshot, but added the more specific assertion below it.